classes/class-wc-connect-taxjar-integration.php

```diff
@@ -262,7 +267,7 @@ public function get_store_settings() {
 			'taxjar_city_setting'     => WC()->countries->get_base_city(),
 		);
 
-		return $store_settings;
+		return apply_filters( 'taxjar_store_settings', $store_settings, $this->settings );
```

Comment:
I'm getting an error on this line:

`
Notice: Undefined property: WC_Connect_TaxJar_Integration::$settings in woocommerce-services/classes/class-wc-connect-taxjar-integration.php on line 270
`

Reply:
My bad, should have tested with `WP_DEBUG` on.

The filter was added [here](https://github.com/taxjar/taxjar-woocommerce-plugin/commit/c8db821cc91536fb0ee6337eb785603b8e76f514).

I fixed it in https://github.com/Automattic/woocommerce-services/pull/1398/commits/73fb20e8134928379d3cb9af1eb872ec13b13d9f, I added a dummy empty array to keep the filter "signature".